### PR TITLE
Schedule

### PR DESCRIPTION
**Pull request by bartosz-gruszczyk**

### CODE REVIEW

**Comment by ziobron** [comment]
✅ CI Passed - Task: schedule-bonus
🏅 Granted 10 XP for @bartosz-gruszczyk

**Comment by ziobron** [comment]
⏱ 3 XP extra for @bartosz-gruszczyk for on-time delivery

**Comment by ziobron** [comment]
✅ CI Passed - Task: schedule
🏅 Granted 12 XP for @bartosz-gruszczyk

**Comment by ziobron** [comment]
⏱ 3 XP extra for @bartosz-gruszczyk for on-time delivery

**Review by lukasz-kukulka — APPROVED** [review]

File: homework/schedule/schedule.hpp [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#pragma once
+#include <chrono>
+#include <iostream>
+#include <thread>
+
+template <typename functionT, typename durationT, typename... arguments>
+auto schedule(functionT func, durationT duration, arguments ...args) {
+    std::this_thread::sleep_for(duration);
+    func(args...);
```
Comment (nauka-programowania-MA) [inline homework/schedule/schedule.hpp:9]: use perfcet forwarding here. std::forward<Args>(args)...

**Review by nauka-programowania-MA — APPROVED** [review]

**Pull request closed without merging**